test/fuzz/transformation_add_type_float_test.cpp

```diff
@@ -61,7 +61,8 @@ TEST(TransformationAddTypeFloatTest, IsApplicable) {
   ASSERT_FALSE(
       transformation.IsApplicable(context.get(), transformation_context));
 
-  // Tests existing 16-bit float type.
+  // The transformation is not applicable because there is already a 16-bit
+  // float type declared in the module
```

Comment:
Nit: the comment you're replacing ended with a period, so end this one with a period too.

Reply:
Done